src/main/java/TaskList.java

```diff
@@ -0,0 +1,78 @@
+import java.util.List;
+
+public class TaskList {
+    List<Task> taskList;
+    Storage storage;
+
+    public TaskList(Storage storage) {
+        this.storage = storage;
+        this.taskList = storage.loadTask();
+    }
+
+    public void addTask(String taskName){
+        try {
+            String taskType = taskName.split(" ", 2)[0];
+            String taskDesc = taskName.split(" ", 2)[1];
+            Task newTask;
+            if (taskType.equals("todo")) {
+                newTask = new ToDo(taskDesc);
+            } else if (taskType.equals("deadline")) {
+                String[] in = taskDesc.split("/",2);
+                newTask = new Deadline(in[0], in[1]);
+            } else {
+                String[] in = taskDesc.split("/", 2);
+                newTask = new Event(in[0], in[1]);
+            }
+            taskList.add(newTask);
+            System.out.println("     Got it. I've added this task:");
+            System.out.println("       " + newTask);
+            System.out.println("     Now you have " + taskList.size() + " tasks in the list.");
```

Comment:
Maybe you can implement a method to print the message instead of printing them one by one?